dimod/higherorder.py

```diff
@@ -23,8 +23,92 @@
 from dimod.binary_quadratic_model import BinaryQuadraticModel
 from dimod.decorators import vartype_argument
 from dimod.vartypes import Vartype
+import dimod
```

Comment:
Things should be imported by name into the namespace rather than using `dimod.`. Makes the import behaviour more consistent.